client/src/components/Home/index.tsx

```diff
@@ -391,6 +407,8 @@ export default class Home extends React.Component<any, IHomeState> {
             defaultValue={template && template.id || ''}
             lineDirection="center"
             placeholder="Choose an ID for the dashboard (will be used in the url)"
+            error={error}
```

Comment:
@deadlyfingers - you're right and what you did with type === 'id' is the best approach.